tests/conftest.py

```diff
@@ -1,8 +1,8 @@
 import pytest
-from responses import RequestsMock
 
 from app.factory import create_app
-
+from flask import url_for
+from responses import RequestsMock
```

Comment:
ok, initially reordered them alphabetically but I guess need to group them first